agreement_maintenance/views/maintenance_equipment_view.xml

```diff
@@ -0,0 +1,47 @@
+<!-- # Copyright (C) 2018 Pavlov Media
+     # Copyright (C) 2019 Open Source Integrators
+     # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="maintenance_equipment_form_view_agreement" model="ir.ui.view">
+        <field name="name">maintenance.equipment.form.agreement</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id"
+               ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <field name="owner_user_id" position="after">
+                <field name="agreement_id"/>
+            </field>
+            <xpath expr="//notebook" position="inside">
+                <page string="Services" id="services">
+                    <field name="serviceprofile_ids"
+                           domain="[('active', '=', True),
+                                    ('equipment_id', '=', False)]"
+                           widget="many2many">
+                        <tree>
+                            <field name="name"/>
+                            <field name="partner_id"/>
```

Comment:
When I try to install the module on a clean v12 instance, I receive an error stating: Field `partner_id` does not exist...
  File "C:\Program Files (x86)\Odoo 12.0e\server\odoo\models.py", line 1112, in _validate_fields
    raise ValidationError("%s\n\n%s" % (_("Error while validating constraint"), tools.ustr(e)))
odoo.tools.convert.ParseError: "Error while validating constraint

Field `partner_id` does not exist
Error context:
View `maintenance.equipment.form.agreement`
[view_id: 525, xml_id: n/a, model: maintenance.equipment, parent_id: 464]
None" while parsing file:/c:/program%20files%20(x86)/odoo%2012.0e/server/odoo/addons/agreement_maintenance/views/maintenance_equipment_view.xml:6, near
<record id="maintenance_equipment_form_view_agreement" model="ir.ui.view">